tests/integration/security/pass_through_filter_chain_test.go

```diff
@@ -258,82 +260,98 @@ spec:
       mode: DISABLE`,
 					expected: []expect{
 						{
-							port:   8085,
-							schema: protocol.HTTP,
+							port:   &echo.Port{ServicePort: 8085, Protocol: protocol.HTTP},
+							schema: scheme.HTTP,
 							want:   false,
 						},
 						{
-							port:   8086,
-							schema: protocol.HTTP,
+							port:   &echo.Port{ServicePort: 8086, Protocol: protocol.HTTP},
```

Comment:
why do we need the protocol twice?

Reply:
I have removed schema, it will be set automatically.